### ISSUE TITLE
@cardano-graphql/cli - TypeError: Cannot read property 'trim' of undefined

### ISSUE THREAD

**Issue by CyberCyclone** [input-output-hk/cardano-graphql#552]
### Environment
```{
 os: {
    arch: 'x64',
    cpus: { count: 12, model: 'AMD Ryzen 9 3900X 12-Core Processor' },
    release: '5.4.0-80-generic',
    totalMem: 24624054272,
    type: 'Linux'
  },
  version: '4.0.0'
}
```

cardano-graphql: 4.0.0

#### Platform

- [X] Linux / Ubuntu (Hyper-V machine on Windows 10)

**Platform version**: 

#### Runtime

- [X] Node.js
- [X] Docker

**Runtime version**: 
```console
node -v: v14.17.2
docker version: 20.10.7
```

### Steps to reproduce the bug
1. Installed with `sudo npm i -g @cardano-graphql/cli@4.0.0`
2. `cd cardano-graphql`
3. `sudo cgql docker snapshot make`. Spits out the below:
```
TypeError: Cannot read property 'trim' of undefined
    at /usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:72:38
    at Array.map (<anonymous>)
    at Object.mapPsOutput (/usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:68:10)
    at Object.<anonymous> (/usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:341:36)
    at step (/usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:44:23)
    at Object.next (/usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:25:53)
    at fulfilled (/usr/lib/node_modules/@cardano-graphql/cli/node_modules/docker-compose/dist/index.js:16:58)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
```

### What is the expected behavior?
Not sure, trying it out for the first time, but was expecting it to write backups of the docker volumes or something.

**Comment by rhyslbw** [comment]
I'm not able to reproduce this with [npx](https://www.npmjs.com/package/npx) 
``` console
npx @cardano-graphql/cli@4.0.0 docker snapshot make
```

```
                      _ 
                     | |
  ____   ____   ____ | |
 / ___) / _  | / _  || |
( (___ ( ( | || | | || |
 \____) \_|| | \_|| ||_|
       (_____|    |_|   
Snapshotting service cardano-db-sync 
./
Snapshotting service cardano-node 
./
Snapshotting service postgres 
./
Snapshot complete 
```

Have you installed the package globally before? If so, try

``` console
npm uninstall -g @cardano-graphql/cli@4.0.0
npm i -g @cardano-graphql/cli@5.0.0
```

**Comment by CyberCyclone** [comment]
It seems to work fine running with npx instead of installing it: `npx @cardano-graphql/cli@4.0.0 docker snapshot make`.

I've stuck with @4.0.0 as that's the cardano-graphql version I'm using currently. Not sure if it's good to mix the versions.

**Comment by CherryDT** [comment]
Same issue here. I don't think it's very discoverable that it only works with `npx` but not with installing it, especially since the readme shows `npm i -g` - but on top of that, `npx` doesn't even work for me!

```
# npx cgql docker rebuild-service
npx: installed 1 in 1.367s
command not found: cgql
```

**Comment by CherryDT** [comment]
Same issue here - but _also_ with `npx`. Can this please be reopened?

**Comment by CherryDT** [comment]
It seems it's because `output` (the result of `docker-compose -q`) is like this:

```
065dc1dae8bd993cd29320b9138051e4869bf992dd1e34c7e8efe57e2eccb6d7
2c07dafe5bd2cca3e2ba788fc6af8a1e52b4876e2d7d72fa499870a13ce5067f
9bbe65e35776f14ef921fd0b4974b364a2b2f90fd8a6dbb0718c3da6349a293b
09767fb95d0e95d8d225c379cea88484c5012e4851915a4bb976917a2e4e25cb
b261e772cd22106827098269f566817e39fa75380131e6a84192b39790636c5a
```

The code then splits this to lines and then splits the lines by 3+ spaces and expects apparently more than one "thing" on the line, but there isn't.

Since the variables are called `nameFragment`, `commandFragment`, etc. I assume you want to call `docker-compose ps` **without** the `-q` switch...

**Comment by CherryDT** [comment]
...I see now this is actually in a dependency, the culprit is this commit: https://github.com/PDMLab/docker-compose/commit/880d2522b5777db30c48569a41dd09fb7e46a4b3

It breaks compatibility with `-q`.

**Comment by AlexZeitler** [comment]
👋 Just wanted to let you know that today we've released version `0.23.15` which should fix this issue.

Looking forward to your feedback!
